doc-content/enterprise-only/installation/kie-server-configure-central-proc.adoc

```diff
@@ -0,0 +1,170 @@
+[id='kie-server-configure-central-proc_{context}']
+= Configuring {KIE_SERVER} to connect to {CENTRAL}
+
+WARNING: This section provides a sample setup that you can use for testing purposes. Some of the values are unsuitable for a production environment, and are marked as such.
+
+If a {KIE_SERVER} is not already configured in your {PRODUCT} environment, or if you require additional {KIE_SERVERS} in your {PRODUCT} environment, you must configure a {KIE_SERVER} to connect to {CENTRAL}.
+
+[NOTE]
+====
+If you are deploying {KIE_SERVER} on {OPENSHIFT}, see the {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_] document for instructions about configuring it to connect to {CENTRAL}.
+====
+
+ifeval::["{context}" == "execution-server"]
+{KIE_SERVER} can be managed or it can be unmanaged. If {KIE_SERVER} is unmanaged, you must manually create and maintain KIE containers (deployment units). If {KIE_SERVER} is managed, the {CONTROLLER} manages the {KIE_SERVER} configuration and you interact with the {CONTROLLER} to create and maintain KIE containers.
+
+[NOTE]
+====
+Only make the changes described in this section if {KIE_SERVER} will be managed by {CENTRAL} and you installed {PRODUCT} from the ZIP files.
```

Comment:
This section contains various admonitions, I'd suggest changing this as normal text. 

```suggestion
Make the changes described in this section if {KIE_SERVER} is managed by {CENTRAL} and you have installed {PRODUCT} from the ZIP files.
```

Reply:
Done